src/SourceBuild/content/eng/tools/join-verticals.proj

```diff
@@ -0,0 +1,46 @@
+<Project>
+    <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+    <PropertyGroup>
+        <!-- Fake, to satisfy the SDK. -->
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    </PropertyGroup>
+
+    <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+    <Target Name="Build">
+        <MSBuild Projects="$(ToolsDir)init-build.proj"
+             Targets="Build"
+             StopOnFirstFailure="true" />
+    </Target>
+
+    <UsingTask TaskName="Microsoft.DotNet.UnifiedBuild.Tasks.JoinVerticals" AssemblyFile="$(MicrosoftDotNetUnifiedBuildTasksAssembly)" />
```

Comment:
This allows to use the in-proc task and executes it out-of-proc which avoid task lock issues when cleaning/rebuilding.

```suggestion
    <UsingTask TaskName="Microsoft.DotNet.UnifiedBuild.Tasks.JoinVerticals" AssemblyFile="$(MicrosoftDotNetUnifiedBuildTasksAssembly)" TaskFactory="TaskHostFactory" />
```